sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Features Added
 
+- Upgrade to API version 2023-07-01-preview
```

Comment:
Please specify the expected release date by replacing the `Unreleased`  with the format `yyyy-mm-dd`.

Reply:
Done.